.github/workflows/release.yaml

```diff
@@ -121,6 +121,34 @@ jobs:
         name: dist-x86_64-unknown-linux-gnu
         path: ./dist
 
+  dist-x86_64-unknown-linux-musl:
+    name: dist (x86_64-unknown-linux-musl)
+    runs-on: ubuntu-20.04
+    env:
+      RA_TARGET: x86_64-unknown-linux-musl
+      # For some reason `-crt-static` is not working for clang without lld
+      RUSTFLAGS: "-C link-arg=-fuse-ld=lld -C target-feature=-crt-static"
+    container:
+      image: rust:alpine
```

Comment:
TIL

Reply:
What does TIL mean though? Non-native speaker here :joy:

Reply:
"Today I Learned" that there's a Rust Alpine Docker image. It's mostly used on Reddit, I guess.